client.proto

```diff
@@ -194,11 +194,46 @@ message ReplToken {
   Permissions permissions = 15;
 }
 
-// GovalTokenMetadata is information about a goval token, that can be used to
+// GovalSigningAuthority is information about a goval token, that can be used to
 // validate it. It is stored in the footer of the PASETO.
-message GovalTokenMetadata {
-  // The ID of the key that was used to sign the token.
-  string key_id = 1;
+message GovalSigningAuthority {
+  oneof cert {
+    // The ID of the public key that was used to sign the token.
+    string key_id = 1;
+
+    // The intermediate_cert that was used to sign the token.
```

Comment:
is this the full contents of the `GovalCert`? if so, might as well make it a `GovalCert` to avoid having this be stringly typed.

Reply:
gotcha! that's an indication that it needs a better comment, haha.